### ISSUE TITLE
Avoiding Detection using Actions

### ISSUE THREAD

**Issue by noodleman420** [ultrafunkamsterdam/undetected-chromedriver#678]
It appears that the actions that I make is detected. For example, navigating to an element and clicking on it.

Is there a better way to make actions like these? It is undetected when I use my mouse manually clicking (or using pyautogui), however once I use an Action, it is detected.

**Comment by sebdelsol** [comment]
You're right sites are getting better at detecting non human reactions time, instant keystrokes, mouse not travelling, etc...
Please check here for some "enhanced" actions I added to [**handle some of those issues**](https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/624#issuecomment-1119872525).

**Comment by noodleman420** [comment]
Thank you @sebdelsol, appreciate it. Do you have any other tips for mouse clicks using Actions?

**Comment by sebdelsol** [comment]
It depends a lot on the targeted site... I've stumbled upon on some that were sensible to too fast reaction or too fast keystrokes, other that timed their sliders, other that expected the mouse to travel through the page before a click... When it doesn't work, rely on common sense : what would an actual human do ?

**Comment by xploader** [comment]
@noodleman420  Have you found any solution so far for mouse movement other than pyautogui?

**Comment by noodleman420** [comment]
@xploader Nope, I believe most of the Selenium actions are detected with this specific antibot I am trying to avoid. I moved to using Playwright in JS for now.
